techniques/systemSettings/misc/variableFromJsonFile/3.0/metadata.xml

```diff
@@ -27,7 +27,7 @@ For example a JSON containing `{ "datacenter": "paris" }` read into the variable
   </TRACKINGVARIABLE>
 
   <SECTIONS>
-    <SECTION name="Base file" component="true" componentKey="VARIABLE_FROM_JSON_FILE_JSON_FILE" />
+    <SECTION name="Base file" component="true" componentKey="VARIABLE_FROM_JSON_FILE_BASE_FILE" />
```

Comment:
i'm dubious that it might work given that the componentKey is not related to the trackingVariable